functionary/core/api/mixins.py

```diff
@@ -29,6 +31,7 @@ class EnvironmentViewMixin:
         ),
     ]
 
+    @cache
```

Comment:
Since this is 3.9+ should we explicitly document our minimum Python version somewhere?

Reply:
I added a small section in the README explaining that the goal is to always be on the latest version of python.